notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -30,7 +30,7 @@
         "id": "fsv4jGuU89rX"
```

Comment:
<p>Several notebooks seem to have this duplicate "Vertex AI Vertex AI" in the title.</p><p> </p><p>Can you do a find-and-replace for "Vertex AI Vertex AI" --&gt; "Vertex AI" across all files?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Fixed.